storage/aptosdb/src/state_store/mod.rs

```diff
@@ -768,9 +783,10 @@ impl StateValueWriter<StateKey, StateValue> for StateStore {
     }
 }
 
-fn add_kv_batch(batch: &mut SchemaBatch, kv_batch: &StateValueBatch) -> Result<()> {
-    for (k, v) in kv_batch {
-        batch.put::<StateValueSchema>(k, v)?;
-    }
+fn add_kv_batch(batch: &SchemaBatch, kv_batch: &StateValueBatch) -> Result<()> {
+    kv_batch
+        .par_iter()
```

Comment:
without min_len?

Reply:
Doesn't work on map iterator..

BTW I moved the commit code path away from this function, to save a clone. This function now is only used by restore path.